api.go

```diff
@@ -373,15 +373,30 @@ func (api *API) delSilence(w http.ResponseWriter, r *http.Request) {
 }
 
 func (api *API) listSilences(w http.ResponseWriter, r *http.Request) {
-	sils, err := api.silences.All()
+	var (
+		num    = r.FormValue("n")
```

Comment:
I pushed a commit that removes the limit from the user's view. Let me know what you think.